### PR TITLE
open-cluster-management/backlog#8222 Add obs 2 qsg

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
@clyang82 and @dockerymick : Can you review this added piece about observability in the Quick Start Guide? Thank you!

**Review by clyang82 — APPROVED** [review]

File: about/quick_start.adoc [hunk 1]
```diff
@@ -57,3 +57,10 @@ See the link:../security/policy_example.adoc#policy-overview[Policy overview] fo
 . From the _Policies_ page, you can view a summary of cluster and policy violations.
 . View your policies from the _Governance and risk_ page in the console.
 You can also view policy details from the cluster _Overview_.
+
+[#observability-quick]
+== Observe clusters
+
+You can enable the observability service to gain insight and optimize your managed clusters. Enable the observability service operator (multicluster-observability-operator) to monitor the health of your managed clusters.
+. Learn more about link:../observability/observe_environments.adoc#observing-environments[Observing environments] and how to link:../observability/enable_observability.adoc#enable-observability[Enable observability service].
```
Comment (dockerymick) [inline about/quick_start.adoc:65]: the operator name should be monospaced

Comment (dockerymick) [inline about/quick_start.adoc:65]: There is an extra period unless you are trying to number the statements

**Review by dockerymick — COMMENTED** [review]
Left a few comments

File: about/quick_start.adoc (hunk 1 above)
Comment (cadawson) [reply about/quick_start.adoc:65]: Added the style.

Comment (cadawson) [reply about/quick_start.adoc:65]: It is numbered, for consistency with the other entries.

**Pull request merged**